code/style.css

```diff
@@ -1,7 +1,551 @@
+  font-family: "Playfair Display", serif;
+  font-size: 18px;
+  font-weight: 500;
+  text-align: center;
+  margin: 30px; /* change the margin for other devices*/
+  padding-left: 50px; /* change the padding for other devices*/
+  padding-right: 50px; /* change the padding for other devices*/
+}
+
+/* SIGN-UP FORM */
+* {
+  box-sizing: border-box;
+}
+label {
+  color: white;
+}
+#sign-up {
+  width: 100%;
+  margin: 0;
+  padding-top: 80px;
+  padding-bottom: 80px;
+  background-color: #deb887; /* For browsers that do not support gradients */
+  background-image: linear-gradient(
+    to right,
+    #f1e1cd,
+    #f6e2c9,
+    #fce3c3,
+    #ffe4bc
+  );
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  flex-flow: wrap;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+section.sign-up-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 100%;
+}
+
+.form-sign-up {
+  background-image: url(./img/orange.jpg);
+  background-repeat: no-repeat;
+  -moz-background-size: cover;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  max-width: 100%;
+  width: 500px;
+  height: 700px;
+  color: white;
+  border-radius: 5px;
+  box-shadow: 5px 15px 55px grey;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 15px;
+  margin-top: 20px;
+}
+.form-sign-up h3 {
+  font-family: "Playfair Display", serif;
+  color: tan;
+  font-size: 50px;
+  letter-spacing: 2px;
+  font-weight: 700;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 10px;
+  justify-content: center;
+}
+.form-sign-up .btn {
+  color: #292524;
+  font-family: "Playfair Display", serif;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 16px;
+  background-color: burlywood;
+  border: 1px solid tan;
+  width: 100%;
+  text-align: center;
+  justify-content: center;
+  outline: none;
+}
+.form-sign-up .btn:hover {
+  color: #ffe4bc;
+  background-color: #292524;
+  letter-spacing: 3px;
+}
+.form-group {
+  font-size: 18px;
+  font-family: "Playfair Display", serif;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  color: black;
+  font-weight: 900;
+}
+/* STYLING INPUTS */
+input {
+  padding: 1px 2px;
+  border: 2px solid tan;
+  border-radius: 10px;
+  background-color: #f1e1cd;
+}
+/* STYLING RADIO BUTTONS */
+.wrapper-row {
+  display: flex;
+  justify-content: flex-start;
+}
+.wrapper-col {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 10%;
+}
+.radiobutton-container {
+  display: block;
+  position: relative;
+  padding-left: 3px;
+  margin: 5px;
+  font-size: 16px;
+  user-select: none;
+}
+.radiobutton-container input {
+  cursor: pointer;
+}
+/* ERROR MESSAGE */
+.error-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  margin: 0;
+}
```

Comment:
Think about spacing between your classes that you're styling - there needs to be some space and air to be more readable!